server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -233,7 +104,31 @@ export function configureWebSocketServices(
 	collaborationSessionEventEmitter?: TypedEventEmitter<ICollaborationSessionEvents>,
 	clusterDrainingChecker?: core.IClusterDrainingChecker,
 ) {
+	const lambdaSettings: INexusLambdaSettings = {
+		ordererManager,
```

Comment:
nit: For future maintainability, maybe we could group actual setting values in INexusLambdaSettings and then for the manager references (ordererManager,clientManager, socketTracker, etc...) have an INexusLambdaDependencies or Services. Just thought it would make things easier to track in the future

Reply:
Good idea!